packages/react-aria-components/src/DropZone.tsx

```diff
@@ -62,12 +62,11 @@ function DropZone(props: DropZoneProps, ref: ForwardedRef<HTMLDivElement>) {
   let stringFormatter = useLocalizedStringFormatter(intlMessages, 'react-aria-components');
 
   let textId = useSlotId();
-  let dropzoneId = useSlotId();
   let ariaLabel = props['aria-label'] || stringFormatter.format('dropzoneLabel');
   let messageId = props['aria-labelledby'];
-  // Chrome + VO will not announce the drop zone's accessible name if useLabels combines an aria-label and aria-labelledby
```

Comment:
apparently this doesn't seem to be an issue anymore? seems like chrome + VO is now happy to announce the accessible name if useLabel combines aria-label and aria-labelledby so i've removed the workaround and updated the tests accordingly

Reply:
we might want to bisect chrome a bit so we don't remove while a recent-ish version still has the issue
can you do that? https://github.com/jay0lee/chrome-bisect